### ISSUE TITLE
xLights integration?

### ISSUE THREAD

**Issue by Puntoboy** [sinedied/dmx-hue#20]
I'm new to this but I have the service running on my PC now, linked to the Hue bridge and I have my lights added.  I've added a new controller on xLights and chosen Artnet as the protocol. Added the IP of my PC. but I can't actually see any lights. Now I could be doing something wrong in xLights but any ideas?

**Comment by swissjon** [comment]
Did you figure out the xlights settings? It seems like I have tried everything.

**Comment by Puntoboy** [comment]
Afraid not.

**Comment by swissjon** [comment]
Bummer, I feel like it should be possible. Did you figure out how to use dmx-hue with another piece of software?

**Comment by sinedied** [comment]
Let me know if you find something that needs to be fixed in dmx-hue. I could only test with QLC+ when I built this tool, but xLights looks interesting and should be working if artnet is supported.

**Comment by swissjon** [comment]
I'm not sure it anything needs to be fixed, I would just like to use this with xlights. xlights definitely supports artnet, I have a feeling I am not configuring xlights correctly to get this work. 

I would love to know if anyone else has been successful at this?

**Comment by star-dust-1** [comment]
Hi! I just got XLights to work with my Hue bulbs yesterday after days of struggling. I'm not an expert by any means, but maybe if I describe my process it could help? 

So I was definitely in the same boat where I could control the lights over QLC+ but got nothing when I tried XLights. The test mode was totally unresponsive and I was losing my mind. 

I have 3 hue lights on my bridge, bulb 1 on channels1-3, bulb 2 on ch4-6, and bulb 3 on ch7-9
![dmx-hue setup](https://user-images.githubusercontent.com/26891371/111053114-fdabb380-8415-11eb-9675-7ef08b181c2f.jpg)


Controller Setup Tab- I set the IP address field to my **local address** (which you can find using the XScanner tool), not the hue bridge's address. The connection dot on the bottom is green so that means it's got an active connection. 
![Controller Setup](https://user-images.githubusercontent.com/26891371/111042960-b5c86480-83f4-11eb-95c9-39d4c009f9c9.jpg)


Layout Setup Tab - For the lights themselves, I think the key is setting the Start Channel field to whatever you've specified your ArtNet controller to be. These are the settings I used for my 3 hue lights:
![Hue Layout Options](https://user-images.githubusercontent.com/26891371/111043632-61bf7f00-83f8-11eb-9da7-4a2067d27108.jpg)

Once I got the settings configured this way, the lights started responding! Hopefully this works for you!

**Comment by swissjon** [comment]
Thanks for posting update. I will give this a try again sometime soon, I have put this project on the backburner for now.

**Comment by star-dust-1** [comment]
Absolutely, I'd love to know if it works for you! I just grabbed a second computer, followed the same setup, and it worked on it as well. One important update to my post (which I edited above) is that you need to set the IP Address to your **Local Address** (which you can find via the xScanner tool). I was mistaken when I said that it was my wifi router's address.

**Comment by thomasrongy** [comment]
Hello, this problem doesn't seem to have been solved.

Have you tried again? I'd like to create a show sequence with xlights but I'm new to DMX. Do you have a complete tutorial on how to do this?
